### PR TITLE
XS✔ ◾ Several improvements to the "checked by" rule

### PR DESCRIPTION
**Pull request by tiagov8**
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

@synnoveisabel 's **Re: Isabel Sandstroem | SSW.People | Australia's Leading .NET and Azure Consultants**

> 2. What was changed?

Added that person who checked should be cc'ed
+ many improvements to the rule readability

> 3. Did you do pair or mob programming (list names)?

no

### CODE REVIEW

File: rules/checked-by-xxx/rule.md [hunk 1]
```diff
@@ -18,40 +18,50 @@ redirects:
...
+* Sending a sensitive email (e.g. The content could be perceived as confrontational)
+* You need a quick '[test please](/conduct-a-test-please)', and the person is immediately available. This provides you a chance to check and collaborate on the finished change before the '[done email](/dones-do-you-reply-done-and-delete-the-original-email)' is sent. It also shows the email was double-checked.
 
 Doing this will ensure:
 
 * You avoid grammatical and spelling errors
+* Your message is clear
 * The layout and tone of the email is professional
-* You haven’t forgotten anything important
+* You don’t forgotten anything important
```
Comment (UlyssesMaclaren) [inline rules/checked-by-xxx/rule.md:50]: Change to 
    You don’t forget anything important

**Review by jaydenalchin — APPROVED** [review]
LGTM pending Uly's suggestion

File: rules/checked-by-xxx/rule.md (hunk 1 above)
Comment (sethdailyssw) [reply rules/checked-by-xxx/rule.md:50]: done

Comment (sethdailyssw) [inline rules/checked-by-xxx/rule.md:50]:
```suggestion
* You don’t forget anything important
```

**Review by sethdailyssw — APPROVED** [review]

**Pull request merged**